schema/string.go

```diff
@@ -17,18 +17,30 @@ const (
 	stringUUIDVersion = 4
 )
 
-func checkStringConstraints(v string, minLength, maxLength int, t string) error {
+func checkStringConstraints(v string, minLength, maxLength int, pattern, t string) error {
```

Comment:
There is no need for the type, here it is always going to be string.

Reply:
ok, I will remove this.